azure-pipelines.yml

```diff
@@ -589,6 +589,32 @@ jobs:
     displayName: Execute tests
   - template: ci/publish-bazel-test-results.yml
 
+- job: execute_sival_fpga_tests_cw310
+  displayName: CW310 SiVal Tests
+  pool:
+    name: $(fpga_pool)
+    demands: BOARD -equals cw310
+  timeoutInMinutes: 30
+  dependsOn:
+    - chip_earlgrey_cw310
+    - sw_build
+  condition: succeeded( 'chip_earlgrey_cw310', 'sw_build' )
+  steps:
+  - template: ci/checkout-template.yml
+  - template: ci/install-package-dependencies.yml
+  - template: ci/download-artifacts-template.yml
+    parameters:
+      downloadPartialBuildBinFrom:
+        - chip_earlgrey_cw310
+        - sw_build
+  - bash: |
+      set -e
+      . util/build_consts.sh
+      module load "xilinx/vivado/$(VIVADO_VERSION)"
+      ci/scripts/run-fpga-tests.sh cw310 fpga_cw310_sival,-manuf,-hyper310 || { res=$?; echo "To reproduce failures locally, follow the instructions at https://opentitan.org/book/doc/getting_started/setup_fpga.html#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```

Comment:
Are you sure these should be running with `cw310` as the interface and not `hyper310`?

Reply:
I think so but I am not 100% certain

Reply:
Yes, they should run on `cw310`. We haven't finished off `hyper310` support yet.

There are two related PRs with different approaches, and we need to sort out how we're going to handle tags:
1. https://github.com/lowRISC/opentitan/pull/20396
2. https://github.com/lowRISC/opentitan/pull/20390

In addition, I debugged some issues in hyperdebug firmware related to JTAG, and we'll need a fix to get in to get all the tests passing:
https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/5061310